redis_views/response.py

```diff
@@ -5,12 +7,22 @@
 
 import redis
 
+logger = logging.getLogger(__name__)
+
 
 class RedisTemplateResponse(TemplateResponse):
 
     def __init__(self, *args, **kwargs):
         super(RedisTemplateResponse, self).__init__(*args, **kwargs)
-        self.connection = redis.StrictRedis.from_url(settings.REDIS_URL)
+        socket_timeout = 2
```

Comment:
I will set it as a constant, which is really the default value if the setting doesn't exist.